### PR TITLE
add a environment variable HOLOVIEWS_INLINE_JS

### PR DESCRIPTION
**Pull request by basnijholt**
Like @philippjfr suggested on chat.

### CODE REVIEW

File: holoviews/ipython/__init__.py [hunk 1]
```diff
@@ -83,7 +83,9 @@ class notebook_extension(extension):
 
     logo = param.Boolean(default=True, doc="Toggles display of HoloViews logo")
 
-    inline = param.Boolean(default=True, doc="""
+    inline = param.Boolean(
+        default=os.environ.get('HOLOVIEWS_INLINE_JS', 'True').lower() == 'true',
```
Comment (philippjfr) [inline holoviews/ipython/__init__.py]: I don't quite follow how this would work: ``'true' != True``

**Comment by philippjfr** [comment]
We've discussed making hv.config configurable via environment variables, so I think I'd prefer going ahead with that plan and moving this option onto ``hv.config``. @jlstevens Any opinions?

**Comment by basnijholt** [comment]
all of the following would result in `inline=True`
```
export HOLOVIEWS_INLINE_JS="TRUE"
export HOLOVIEWS_INLINE_JS="true"
export HOLOVIEWS_INLINE_JS="True"
```
otherwise this would be `False`.

**Comment by jlstevens** [comment]
I agree with @philippjfr - this should be controlled through `hv.config` and that would be the most sensible thing to control via environment variables.

**Comment by philippjfr** [comment]
Need to decide how that will work. In panel I've used parameters and property pairs to make config options that are also controllable by environment variables, but I'm also okay if we take the simpler approach of simply setting the value on import like in this PR is fine too.

**Comment by philippjfr** [comment]
This is no longer needed, we have switched back to inlining JS by default.

**Pull request closed without merging**